test/e2e/kubelet_perf.go

```diff
@@ -208,16 +205,30 @@ var _ = Describe("Kubelet [Serial] [Slow]", func() {
 		// noise.
 		rTests := []resourceTest{
 			{podsPerNode: 0,
-				limits: containersCPUSummary{
+				cpuLimits: containersCPUSummary{
 					"/kubelet":       {0.50: 0.06, 0.95: 0.08},
 					"/docker-daemon": {0.50: 0.05, 0.95: 0.06},
 				},
+				// We set the memory limits generously because the distribution
+				// of the addon pods affect the memory usage on each node.
+				memLimits: resourceUsagePerContainer{
+					"/kubelet":       &containerResourceUsage{MemoryRSSInBytes: 70 * 1024 * 1024},
+					"/docker-daemon": &containerResourceUsage{MemoryRSSInBytes: 85 * 1024 * 1024},
+				},
 			},
+			// TODO(yujuhong): change this test to ~100 pods per node after
+			// --max-pods have been changed.
 			{podsPerNode: 35,
```

Comment:
Should we update the test to run `100` pods? Since the limit is lesser than that as of now, we can do it in a separate PR.